diagnostic.md

```diff
@@ -48,11 +50,11 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    Add it to the model? I dont really understand the question.
     ```
```

Comment:
Correct that the model would handle this within the Route object. The model can take a parameter of `params`, which provides all route params for us. Thus, to get the id of 123 in the Route object (aka the model of the Route object), we'd use `params.movie_id`